modules/ROOT/pages/deployment/container/orchestration/tab-pages/breaking-changes.adoc

```diff
@@ -0,0 +1,62 @@
+= Helm Chart Breaking Changes
+:toc: right
+
+////
+Note that there is only this one master file for all breaking changes.
+For each breaking changes block you need:
+* a new section on top of the lastest one. (actual first, former second)
+* directly above the section an ID (see below how it is done) with the corresponding version equal to the version name so it can be accessed via an anchor. This cannot be automated and must be maintained manually.
+* Even if there are no breaking changes, add a section telling that there are no breaking changes.
+* You cannot substitute the version with an attribute like {helm_tab_2_tab_text} as the attribute changes and the content here needs to be static.
+* You can, if wanted, delete all sections if there are more than two except the last two which are mandatory to be present because of referencing.
```

Comment:
```suggestion
* If there are more than two sections, you can delete some or all except the last two which are mandatory to be present because of referencing.
```